src/repl.ts

```diff
@@ -759,7 +769,7 @@ export function activate(context: vscode.ExtensionContext, settings: settings.IS
 
     context.subscriptions.push(vscode.commands.registerCommand('language-julia.executeJuliaCodeInREPL', executeSelection));
 
-    context.subscriptions.push(vscode.commands.registerCommand('language-julia.executeJuliaFileInREPL', executeFile));
+    context.subscriptions.push(vscode.commands.registerCommand('language-julia.executeJuliaFileInREPL', (uri?: vscode.Uri) => { executeFile(uri) }));
```

Comment:
Do we need that change here? I would have thought that we don't need to introduce the anonymous function here, but can just continue to pass `executeFile`?

Reply:
I think that's the only way to get the path to the file that was selected in the explorer. Got the answer from here: https://stackoverflow.com/questions/51961457/how-to-get-file-name-or-path-in-vscode-extension-when-user-right-click-on-file-i/

Reply:
It just seems to me that the expression `executeFile` and `(uri?: vscode.Uri) => { executeFile(uri) }` in both cases evaluates into a function that has the same signature? Could you try the other option? If it doesn't work, happy to go with what is here.

Reply:
You are right. I checked again and everything seems to work fine without the anonymous function.